spark-task-sender/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM golang
```

Comment:
I'll take of it after merging the code. Basically we need to compile the go code and completely avoid the dependency from Go.